ci-operator/step-registry/cert-manager/clusterissuer/cert-manager-clusterissuer-commands.sh

```diff
@@ -4,49 +4,65 @@ set -e
 set -u
 set -o pipefail
 
+if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
+    source "${SHARED_DIR}/proxy-conf.sh"
+    echo "proxy: ${SHARED_DIR}/proxy-conf.sh"
+fi
+
+timestamp() {
+    date -u --rfc-3339=seconds
+}
+
+# Define common variables
+SUB="openshift-cert-manager-operator"
+OPERATOR_NAMESPACE="cert-manager-operator"
+OPERAND_NAMESPACE="cert-manager"
+CLUSTERISSUER_NAME="cluster-certs-clusterissuer" # This clusterissuer is consumed by the 'cert-manager-custom-apiserver-cert' and 'cert-manager-custom-ingress-cert' refs.
+INTERVAL=10
+
 function wait_cert_manager_rollout() {
     local OLD_POD=$1
     local MAX_RETRY=12
-    local INTERVAL=10
     local COUNTER=0
     while :;
     do
-        NEW_POD_OUTPUT=$(oc get po -l app.kubernetes.io/name=cert-manager -n cert-manager)
+        echo "# Checking the cert-manager controller pod status for the #${COUNTER}-th time ..."
+        NEW_POD_OUTPUT=$(oc get po -l app.kubernetes.io/name=cert-manager -n $OPERAND_NAMESPACE)
         if [[ ! "$NEW_POD_OUTPUT" =~ $OLD_POD ]] && [[ "$NEW_POD_OUTPUT" == *1/1*Running* ]]; then
-            echo "The cert-manager pod finished rollout." && break
+            echo "[$(timestamp)] Finished the cert-manager controller pod rollout."
+            break
         fi
         ((++COUNTER))
         if [[ $COUNTER -eq $MAX_RETRY ]]; then
-            echo "The cert-manager pod still does not finish rollout after $((MAX_RETRY * INTERVAL)) seconds. Dumping status:"
-            oc get po -n cert-manager
+            echo "[$(timestamp)] The cert-manager controller pod didn't finish rollout after $((MAX_RETRY * INTERVAL)) seconds. Dumping status:"
+            oc get po -n $OPERAND_NAMESPACE
             exit 1
         fi
         sleep $INTERVAL
     done
 }
 
-if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
-    source "${SHARED_DIR}/proxy-conf.sh"
-    echo "proxy: ${SHARED_DIR}/proxy-conf.sh"
-fi
+function prepare_cloud_credentials() {
```

Comment:
@lunarwhite PR LGTM. If named `configure_cloud_credentials`, it is more precise, because actually only `oc create -f - <<< "${MANIFEST}"` does the `prepare` work, additional `patch`es' work is using the `prepare`d credentials to do the work of `configure` cert-manager.

Reply:
Updated with the naming suggestion. It indeed sounds better now!

Also added a separate commit to update the jobs suffix name, just for consistency.